src/components/DocArrowNavigators.js

```diff
@@ -22,15 +22,19 @@ const Styled = styled.div`
     }
 `
 
-const DocArrowNavigators = ({prev, next}) => (
+const DocArrowNavigators = ({prev, next, prevTitle, nextTitle}) => {
+    const pTitle = `Previous Page ${prevTitle ? `: ${prevTitle}` : ""}`
+    const nTitle = `Next Page ${nextTitle ? `: ${nextTitle}` : ""}`
```

Comment:
If you feel like there is room for a few more words, I think an explicit verb could remove any ambiguity as to what this link is for.

If it's already long enough, disregard this suggestion, as I think it's already pretty obvious what the link does.

```suggestion
    const nTitle = `Go to next page ${nextTitle ? `: ${nextTitle}` : ""}`
```